types/shuffle-array/index.d.ts

```diff
@@ -1,22 +1,23 @@
-// Type definitions for shuffle-array
+// Type definitions for shuffle-array 1.0.1
```

Comment:
This should only be major and minor. If you enable the lint rule in dtlint.json it should catch this (`dt-header` IIRC).

Reply:
Thank you for your feedback!

I updated the version info to `1.0´.

About the `dt-header` rule, I saw that it's disabled in shuffle-array's tslint.json. I enabled it and had to change the first contributor's URL (the dt-header rule seems to allow only github URLs in the header). I would say that's fine since it is evidently the correct github profile and a link to their personal website can be found there.